src/_guides/language/type-system.md

```diff
@@ -466,26 +460,36 @@ DartPad: {{site.dartpad}}/4a2a9bc2242042ba5338533d091213c0
 [Try it in DartPad]({{site.dartpad}}/4a2a9bc2242042ba5338533d091213c0).
 {% endcomment %}
 
-What about going in the other direction? Can you assign an `Animal` list to a `List<Cat>`?
+What about going in the other direction? 
+Can you assign an `Animal` list to a `List<Cat>`?
 
 {:.passes-sa}
-<?code-excerpt "strong/lib/strong_analysis.dart (generic-type-assignment-Animal)" replace="/Animal/[!$&!]/g"?>
+<?code-excerpt "lib/strong_analysis.dart (generic-type-assignment-Animal)" replace="/Animal/[!$&!]/g"?>
 {% prettify dart tag=pre+code %}
 List<Cat> myCats = <[!Animal!]>[];
 {% endprettify %}
 
-This assignment passes static analysis,
-but it creates an implicit cast. It is equivalent to:
+This assignment does not pass static analysis 
+as it creates an implicit downcast, 
+which are disallowed from non-`dynamic` types such as `Animal`.
```

Comment:
```suggestion
This assignment doesn't pass static analysis 
because it creates an implicit downcast, 
which is disallowed from non-`dynamic` types such as `Animal`.
```
https://developers.google.com/style/contractions
https://developers.google.com/style/word-list#as